esmvaltool/diag_scripts/zmnam/zmnam_preproc.py

```diff
@@ -8,11 +8,20 @@
 import cdo as cd
 
 
-def zmnam_preproc(ifile):
+def zmnam_preproc(ifile, hemisphere):
     """Preprocessing of the input dataset files."""
     cdo = cd.Cdo()
+
+    # Select hemisphere depending on recipe parameter
+    if hemisphere == 'NH':
```

Comment:
Yes, in fact I was thinking to include information on the hemisphere in both the netCDF metadata and the plots.

Reply:
Actually it is needed to report the information in the output files if two hemispheres can be selected. This is now done!

Reply:
I preferred to leave the possibility to select just one or both hemispheres from the recipe setting. The processing time is not too different when selecting just one hemisphere but the number of plots is smaller and this may be useful when considering many datasets.

Reply:
Thanks, looks good now!